### ISSUE TITLE
Edit/delete project

### ISSUE THREAD

**Issue by Playmojs** [Playmojs/fantasy-mapbuilder_2#21]
Should probably take users to a unique project home-page, in which project-name, the main project-map (as well as the project image?) and the users with accessibility to the project is available. It should also be determined who should have access to this page, and how this access can be restricted (because read-information is readily given from all projects).
